src/components/AddAlbum.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import { addAlbum } from "@/actions";
+import { Genre, genreEnum } from "@/db/schema";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import toast from "react-hot-toast";
+
+export default function AddAlbum() {
+  const [title, setTitle] = useState("");
+  const [artist, setArtist] = useState("");
+  const [genre, setGenre] = useState("");
+  const genres = genreEnum.enumValues;
+  const router = useRouter();
+
+  async function handleAdd() {
+    if (title.length < 3 || artist.length < 3) {
+      toast.error(
+        "please enter valid album, inputs cannot be shorter than 3 lol",
+      );
+    } else {
+      try {
+        await addAlbum({ title, artist, genre });
+      } catch (error) {
+        console.error(error);
+      }
+    }
+    router.refresh();
```

Comment:
You always refresh the route even if an invalid configuration was given, which looks kind of weird when using the page, as you try to refresh although it is not needed.